src/Test.UnitTests.Sarif/Test.UnitTests.Sarif.csproj

```diff
@@ -50,9 +50,11 @@
     <None Remove="TestData\SarifVersionOneToCurrentVisitor\ExpectedOutputs\MinimumWithLanguage.sarif" />
     <None Remove="TestData\SarifVersionOneToCurrentVisitor\ExpectedOutputs\NestedInnerExceptionsInNotifications.sarif" />
     <None Remove="TestData\SarifVersionOneToCurrentVisitor\ExpectedOutputs\OneRunWithAllReportingDescriptors.sarif" />
+    <None Remove="TestData\SarifVersionOneToCurrentVisitor\ExpectedOutputs\WebScout.sarif" />
```

Comment:
>WebScout [](start = 75, length = 8)

This is an internal tool. I suggest:

1. Rename it to something like V1ZeroLengthLocationsArray.sarif
2. Strip it down to the absolute minimum file that demonstrates the problem. One result, no optional properties... just what you need to trigger the bug.

Reply:
Ok! will update!

---
In reply to: [292206492](https://github.com/microsoft/sarif-sdk/pull/1527#discussion_r292206492) [](ancestors = 292206492)